build.sbt

```diff
@@ -0,0 +1,192 @@
+import com.typesafe.sbt.SbtNativePackager.autoImport.NativePackagerHelper._
+import com.typesafe.sbt.packager.docker.DockerPlugin.autoImport.{Docker, dockerRepository}
+import com.typesafe.sbt.packager.docker.{Cmd, ExecCmd}
+import org.knora.Dependencies
+import sbt.Keys.version
+
+import scala.language.postfixOps
+import scala.sys.process.Process
+
+//////////////////////////////////////
+// GLOBAL SETTINGS
+//////////////////////////////////////
+
+lazy val aggregatedProjects: Seq[ProjectReference] = Seq(webapi)
```

Comment:
do we need this aggregation, if it's only one element in it?

Reply:
I think we are going to add more projects in new steps, therefore I leave it.